db/rules_of_origin/mocked_schemes.json

```diff
@@ -0,0 +1,26 @@
+{
+  "scope": "uk",
+  "links": [
+    {
+      "text": "Gov UK",
+      "url": "https://www.gov.uk/"
+    },
+    {
+      "text": "HMRC",
+      "url": "http://www.hmrc.gov.uk/"
+    }
+  ],
+  "schemes": [
+    {
+      "scheme_code": "EU",
+      "title": "Mocked scheme data",
+      "countries": [
+        "FR",
+        "ES",
+        "IT"
+      ],
+      "footnote": "This is a mocked footnote"
+    }
+  ]
+}
+
```

Comment:
this is not `jsonapi` isn't it?
just asking, maybe don't want to use it here

Reply:
This is a mocked version of a json file in the system, schema is of Matt's designing